subsys/bluetooth/host/Kconfig

```diff
@@ -337,6 +340,7 @@ config BT_SMP
 if BT_SMP
 config BT_PRIVACY
 	bool "Privacy Feature"
+	select BT_HAS_CENTRAL_ADDRESS_RESOLUTION if BT_CENTRAL
```

Comment:
I'd prefer to have the logic near the symbol itself (ie using `def_bool`). No strong opinion since you're selecting itfrom the same file though.